design-proposals/klip-42-schema-migrations-tool.md

```diff
@@ -363,30 +370,33 @@ Commands
               
               If 'all' is specified, then it migrates all newer versions available
               If 'next' is specified, then it migrates only the next available version
-              If 'until <target>' is specified, then it migrates all available versions before <target>
+              If 'until <target>' is specified, then it migrates all available versions up to and including <target>
   
   info        Displays information about the current and available migrations
   
-  baseline    Sets the current schema to the specified version
```

Comment:
This removal is based on the conversation in https://github.com/confluentinc/ksql/pull/6721#discussion_r545181281